api_intro.py

```diff
@@ -1,24 +1,31 @@
 from datetime import date
+import json
 
 import requests
 
-import config
+# Get configuration data from config.json
+try:
+    with open("config.json", "r") as config_file:
+        config_data = json.load(config_file)
+        config_file.close()
+except FileNotFoundError:
+    raise Exception('Config.json file is missing.')
 
-# Client IDs and Secrets from config.py
+# load configuration constants
 try:
-    OB_CLIENT_ID = config.OB_CLIENT_ID
-    OB_CLIENT_SECRET = config.OB_CLIENT_SECRET
-    SN_CLIENT_ID = config.SN_CLIENT_ID
-    SN_CLIENT_SECRET = config.SN_CLIENT_SECRET
-    BASE_URL = config.BASE_URL
-except AttributeError:
+    OB_CLIENT_ID = config_data['OB_CLIENT_ID']
+    OB_CLIENT_SECRET = config_data['OB_CLIENT_SECRET']
+    SN_CLIENT_ID = config_data['SN_CLIENT_ID']
+    SN_CLIENT_SECRET = config_data['SN_CLIENT_SECRET']
+    BASE_URL = config_data['BASE_URL']
+except KeyError:
     raise Exception('Config file is missing configuration.')
```

Comment:
If getting `KeyError `, I think it's better to tell your users which one is missing.

Reply:
This has been added to the exceptions for the configuration.